python/mlx/nn/layers/activations.py

```diff
@@ -375,4 +549,16 @@ def __call__(self, x: mx.array):
 
 @_make_activation_module(selu)
 class SELU(Module):
-    pass
+    r"""Applies the Scaled Exponential Linear Unit.
+        Applies :math:`\lambda x` if :math:`x > 0` and
+        :math:`\lambda \alpha (\exp(x) - 1)` if :math:`x \leq 0`, where
+        :math:`\lambda = 1.0507` and :math:`\alpha = 1.67326`.
+
+    See :func:`selu`, for the functional equivalent.
+    """
```

Comment:
The decorator `_make_activation_module` actually implement `__init__` and `__call__`. But I think you can add the doc string and remove the implementation.

```python
_make_activation_module(selu)
 class SELU(Module):
     r"""Applies the Scaled Exponential Linear Unit.
         Applies :math:`\lambda x` if :math:`x > 0` and
         :math:`\lambda \alpha (\exp(x) - 1)` if :math:`x \leq 0`, where
         :math:`\lambda = 1.0507` and :math:`\alpha = 1.67326`.
     See :func:`selu`, for the functional equivalent.
     """
```

Reply:
Unless you have customized `__init__` or `__call__`. Then you remove the decoration and implement the function. You can find similar definitions in the file.